timesheet_automatic_reminder/models/timesheet_reminder.py

```diff
@@ -0,0 +1,333 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime, timedelta
+from collections import defaultdict
+
+from openerp import models, api, fields
+from openerp.tools import DEFAULT_SERVER_TIME_FORMAT as TIME_FORMAT, \
+    DEFAULT_SERVER_DATE_FORMAT as DATE_FORMAT, \
+    DEFAULT_SERVER_DATETIME_FORMAT as DATETIME_FORMAT
+
+
+class RemindDateLine(models.TransientModel):
+    _name = 'reminder.date.line'
+    _description = 'Date need reminder lines'
+
+    # columns
+    reminder_date = fields.Date('Reminder Date')
+    leave_hours = fields.Float('Leave Hours')
+    work_hours = fields.Float('Work Hours')
+    lack_hours = fields.Float('Lack Hours')
+    reminder_id = fields.Many2one('timesheet.reminder', 'Reminder Timesheet',
```

Comment:
@dongshukai General style is this https://github.com/Elico-Corp/maintainer-tools/blob/master/CONTRIBUTING.md#idioms

```
partner_id = fields.Many2one(
    "res.partner",
    "Partner",
    "Required",
)
```

even just one parameter.

On this PR check the others cases and fix it please